### PR TITLE
feat: add Go Plugin Runner docs

### PR DESCRIPTION
**Pull request by spacewander**

### CODE REVIEW

File: website/docusaurus.config.js [hunk 1]
```diff
@@ -345,6 +377,10 @@ module.exports = {
               label: "Apache™️ APISIX Java Plugin Runner",
               to: "/docs/java-plugin-runner/development/"
             },
+            {
+              label: "Apache™️ APISIX JGo Runner",
```
Comment (juzhiyuan) [inline website/docusaurus.config.js]:
```suggestion
              label: "Apache™️ APISIX Go Runner",
```

**Review by juzhiyuan — APPROVED** [review]

File: website/docusaurus.config.js [hunk 2]
```diff
@@ -81,6 +81,17 @@ module.exports = {
         version: "0.1.0",
         releaseDate: "2021-06-23",
         firstDocPath: "/development",
+      },
+      {
+        name: "APISIX™ Go Plugin Runner",
+        nameInParamCase: "go-plugin-runner",
+        description: "Runs Apache APISIX plugins written in Go. Implemented as a sidecar that accompanies Apache APISIX.",
+        shape: "star",
```
Comment (iamayushdas) [inline website/docusaurus.config.js]: changing the shape will be better, not urgent I can do that on next PR

**Review by iamayushdas — APPROVED** [review]

File: website/docusaurus.config.js (hunk 2 above)
Comment (spacewander) [reply website/docusaurus.config.js]: @iamayushdas 
What shape do you recommend?

Comment (iamayushdas) [reply website/docusaurus.config.js]: nonagon

Comment (spacewander) [reply website/docusaurus.config.js]: Updated.

Comment (iamayushdas) [reply website/docusaurus.config.js]: no its not, sir you have to add a shape(nonagon).png to reflect it on doc page

Comment (iamayushdas) [reply website/docusaurus.config.js]: image

Comment (juzhiyuan) [reply website/docusaurus.config.js]: It's difficult for contributors who are not familiar with those configurations, I think it's ok to use Star here, and @iamayushdas you could help to submit a new PR to update the shape.

Comment (iamayushdas) [reply website/docusaurus.config.js]: okay sir

**Review by Yiyiyimu — APPROVED** [review]

File: website/docusaurus.config.js (hunk 2 above)
Comment (iamayushdas) [reply website/docusaurus.config.js]: #404

**Pull request merged**